supplements/acquisitions-incorporated/items.xml

```diff
@@ -81,7 +81,9 @@
 		</setters>
```

Comment:
Kon, can you fix typo while you here, on the line 69, there's dot instead of comma in
`+1 bonus to Strength saving throws. and you have darkvision`

Reply:
done